### PR TITLE
test(config): testing config js file

### PR DESCRIPTION
**Pull request by julienfroidefond**

### CODE REVIEW

File: test/popup.test.js [hunk 1]
```diff
@@ -10,21 +10,26 @@ import { assert } from "chai";
...
-});
-
 describe("popup.js", () => {
-  before(function () {
+  before(() => {
     chrome.extension.connect = () => {};
     global.chrome = chrome;
     global.window = dom.window;
+
+    sandbox.stub(configUtils, "load").returns({ isConfigured: false });
```
Comment (dalexanco) [inline test/popup.test.js:22]: sandbox won't break futur import of configUtils ;)

**Review by dalexanco — APPROVED** [review]

**Comment by dalexanco** [comment]
I add a commit around config.js :)
(no problem if you prefer to discard them)

File: test/utils.config.test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { load, save, clear, defaultConfig } from "../src/js/utils/config";
+import { assert } from "chai";
+
+const spy = { req: null };
+
+const mockedChrome = {
```
[the remark this answers is not in the document]
Comment (julienfroidefond) [reply test/utils.config.test.js]: Yes; I found it overkill for this file

File: test/utils.config.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { load, save, clear, defaultConfig } from "../src/js/utils/config";
```
[the remark this answers is not in the document]
Comment (julienfroidefond) [reply test/utils.config.test.js]: mocha can't find it if I do so

[the remark this answers is not in the document]
Comment (julienfroidefond) [reply test/utils.config.test.js]: I have to check why in mocha this is the case

**Pull request merged**